pkg/bootstrap/docker/up.go

```diff
@@ -84,11 +85,11 @@ var (
 	  %[1]s --host-data-dir=/mydata --use-existing-config
 
 	  # Use a different set of images
-	  %[1]s --image="registry.example.com/origin" --version="v1.1"`)
+	  %[1]s --image="registry.example.com/origin" --version="v1.1"
+
+	  # Use a different set of image streams
+	  %[1]s --image-streams="name:relative/path/to/image-streams.json"`)
 
-	imageStreamLocations = map[string]string{
-		"origin centos7 image streams": "examples/image-streams/image-streams-centos7.json",
-	}
```

Comment:
Should we not provide a default set of image stream locations, and then clear out and replace this map with whatever is provided by `--image-streams` ?